ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -36,25 +52,66 @@ export const NotebookList = withCurrentWorkspace()(class extends React.Component
     notebookList: FileDetail[],
     notebookNameList: string[],
     creating: boolean,
-    loading: boolean
+    loading: boolean,
+    notebookTransferCheckInterval: number,
```

Comment:
Open for suggestion for notebookTransferCheckInterval and showWaitingForNotebookTransferMsg name